Colab RDP/Colab RDP.ipynb

```diff
@@ -297,10 +297,10 @@
         "\n",
         "if Sleep:\n",
         "    from time import sleep\n",
-        "    sleep(43200)"
+        "    sleep(99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999)"
```

Comment:
I believe `43200` is enough because Colab won't allow you to use it for more than 12 hours.